Source/Scene/Cesium3DTileset.js

```diff
@@ -1316,17 +1316,18 @@ Object.defineProperties(Cesium3DTileset.prototype, {
   },
 
   /**
-   * A custom shader to apply to the tileset. Only used for contents that use
-   * {@link ModelExperimental}. Using custom shaders with a {@link Cesium3DTileStyle}
-   * may lead to undefined behavior.
+   * A custom shader to apply to all tiles in the tileset. Only used for
```

Comment:
Maybe add a reference to `ExperimentalFeatures` here, so people know to enable it.